docs/google/specification.md

```diff
@@ -92,15 +92,10 @@ For instance, "Do not apply two CZ gates in a row."
 
 ## Serializable Devices
```

Comment:
title doesn't match now

Reply:
Fixed in d7ae40943eabef5cdf07b6aaea0cff665418a098.